### DISCUSSION TITLE
deactivating logging of api errors

### DISCUSSION THREAD

**Discussion by prstwo** [nuxt/nuxt#20724] [General]
Hello, 
I wonder how to deactivate the logs of response errors in fetch requests. I'm using $fetch hook to fetch data from api. 
here is an example of a log:
`[nitro] [dev] [unhandledRejection] FetchError: 400 Bad Request (https://api.faresbazar.com/api-frontend/Customer/Info)                       16:19:07
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)`

**Comment by TheAlexLichter** [comment]
You can use try/catch around it and handle the error as you want in the `catch` block (e.g. do nothing)

**Reply by prstwo** [reply]
Actually, I did so, but  the problem still exists

**Reply by TheAlexLichter** [reply]
Please provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction) then 🙏

**Reply by prstwo** [reply]
here is the production:https://stackblitz.com/edit/github-wzvezd?file=middleware/checkKey.global.js

**Reply by maou-shonen** [reply]
> here is the production:https://stackblitz.com/edit/github-wzvezd?file=middleware/checkKey.global.js

because you are not using await.

**Reply by prstwo** [reply]
Thanks
